lib/oli_web/templates/static_page/index_logged_in.html.heex

```diff
@@ -24,9 +24,12 @@
           </div>
         </div>
         <%= unless Oli.Accounts.is_lms_user?(@conn.assigns.current_user.email) do %>
-          <div class="flex justify-center max-w-max items-center mt-12 gap-x-1 border-white border-b">
-            <.link class="flex items-center gap-x-1" navigate={~p"/workspaces/student"}>
-              <div class="text-white text-xl font-normal font-['Inter'] leading-normal">
+          <div class="flex justify-center max-w-max items-center mt-12 gap-x-1 border-white border-b hover:border-zinc-300">
+            <.link
```

Comment:
And if we define the text color and hover:text color at the link level, the hover will work "as a group" (no matter if we hover on the text or on the arrow icon)

```suggestion
             <.link
              class="flex items-center gap-x-1 no-underline hover:no-underline text-white hover:text-zinc-300"
              navigate={~p"/workspaces/student"}
            >
              <div class="text-xl font-normal font-['Inter'] leading-normal">
                Access my courses
              </div>
              <div>
                <%= OliWeb.Icons.right_arrow_login(%{}) %>
              </div>
            </.link>
```